### PR TITLE
Add example how to work with VM

### PR DESCRIPTION
**Pull request by lukas-bednar**
Initial patch for #8

Signed-off-by: Lukas Bednar <lbednar@redhat.com>

### CODE REVIEW

**Comment by lukas-bednar** [comment]
I will add some README how to run examples.

**Comment by rmohr** [comment]
@lukas-bednar, ideas on how to test them? I think we should get out of the current trial-and-error situation in the python sdk. E.g. use the examples as mini-tests to verify basic functionality as part of ci in kubevirt/kubevirt. What do you think?

File: examples/example_vms.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    vm_body = read_yaml_file("vm-ephemeral.yaml")
+    pprint(api.create_namespaced_virtual_machine_0(vm_body, NAMESPACE))
+
+    # Wait until VM is running
+    def event_source(**kw):
+        if '_request_timeout' not in kw:
+            kw['_request_timeout'] = 60
+        return api.list_namespaced_virtual_machine_0(NAMESPACE, **kw)
+
+    w = watch.Watch()
```
Comment (pkliczewski) [inline examples/example_vms.py]: We could extract this wait on running vm to separate method. It would make the code a bit clearer

Comment (lukas-bednar) [reply examples/example_vms.py]: Yes, I will move it to `examplelib.py`, thanks

**Comment by lukas-bednar** [comment]
> @lukas-bednar, ideas on how to test them? I think we should get out of the current trial-and-error situation in the python sdk. E.g. use the examples as mini-tests to verify basic functionality as part of ci in kubevirt/kubevirt. What do you think?

@rmohr I think it is good idea. I will take a look how can we do that.

**Pull request merged**